src/main/code.ts

```diff
@@ -116,7 +119,7 @@ if (figma.currentPage.selection.length != 1) {
   
   function getFirstChildOfTypeText(sceneNode: SceneNode) {
 	if ("children" in sceneNode) {
```

Comment:
Since property `children` is not used any more, this check is no longer necessary. 

Moreover, I think we can now inline the call to method `getFirstChildOfTypeText`.

Reply:
The function "findOne()" can only be used when the scene node has children. Therefore the if-else-statement is still necessary.

Reply:
They don't say that in the [docs](https://www.figma.com/plugin-docs/api/properties/nodes-findone/) – you probably found this by experiment?